lms/static/scripts/frontend_apps/components/dashboard/OrderableActivityTable.tsx

```diff
@@ -28,9 +28,14 @@ export type OrderableActivityTableProps<T> = Pick<
  * recent values first.
  */
 const descendingOrderColumns: readonly string[] = [
+  // Annotation metrics
   'last_activity',
   'annotations',
   'replies',
+
+  // Course metrics
+  'last_launched',
+  'assignments',
```

Comment:
This is now dealing with implementation details from consumers of the `OrderableActivityTable`, so I'll probably move this logic to them instead.

Out of the scope of this PR though.